<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Game Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>398</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Window</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3,0,0,0">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Window Size : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="winSizeComboBox">
                  <property name="currentIndex">
                   <number>4</number>
                  </property>
                  <item>
                   <property name="text">
                    <string/>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1366 x 768</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1360 x 768</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1280 x 768</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1280 x 720</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1024 x 768</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>800 x 600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>720 x 576</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>720 x 480</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>640 x 480</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>320 x 240</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>240 x 320</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="winWidthSpinBox">
                  <property name="minimum">
                   <number>20</number>
                  </property>
                  <property name="maximum">
                   <number>9000</number>
                  </property>
                  <property name="value">
                   <number>1280</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="winHeightSpinBox">
                  <property name="minimum">
                   <number>20</number>
                  </property>
                  <property name="maximum">
                   <number>9000</number>
                  </property>
                  <property name="value">
                   <number>720</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="7,0,0,0">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Scene Size (Game World size) : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sceneWidthSpinBox">
                  <property name="minimum">
                   <number>20</number>
                  </property>
                  <property name="maximum">
                   <number>1000000</number>
                  </property>
                  <property name="value">
                   <number>1280</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sceneHeightSpinBox">
                  <property name="minimum">
                   <number>20</number>
                  </property>
                  <property name="maximum">
                   <number>1000000</number>
                  </property>
                  <property name="value">
                   <number>720</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Window Mode :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="winModeComboBox">
                  <item>
                   <property name="text">
                    <string>Window</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Full Screen</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Window Title :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="winTitleLineEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="9,0,0">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Icon File :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="iconFilelabel">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="iconBrowseBtn">
                  <property name="text">
                   <string>Browse</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="9,0">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Frame Rate :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="frameRateDSpinBox">
                  <property name="maximum">
                   <double>240.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>60.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QCheckBox" name="showMouseCheckBox">
                  <property name="text">
                   <string>Show Mouse</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="escapeCheckBox">
                  <property name="text">
                   <string>Use ESC key to exit game</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QCheckBox" name="suspendCheckBox">
                  <property name="text">
                   <string>Suspend game if lost focus</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="motionCompCheckBox">
                  <property name="text">
                   <string>Motion Compensation</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Game background color : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="sceneBGColorBtn">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Audio</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="9,0,0">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Audio Format : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="audioFormatsComboBox">
                  <item>
                   <property name="text">
                    <string>44100 Hz</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>22050 Hz</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>11050 Hz</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="audioChannelsComboBox">
                  <item>
                   <property name="text">
                    <string>Stereo</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Mono</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="9,0">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Maximum number of simultaneous sounds :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="maxAudioSpinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>16</number>
                  </property>
                  <property name="value">
                   <number>8</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Preferences</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
